app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -375,7 +375,6 @@ protected void onCreate(Bundle savedInstanceState) {
         initializeSpeechRecognizer();
 
         mPoorPerformanceAllowList = new HashSet<>();
-        checkForCrash();
```

Comment:
I was also thinking what about keeping the call here and add an unconditional return at the beginning of the method with a comment explaining that will be enabled later once we have the required infra?